### ISSUE TITLE
Question about ubi8-micro example

### ISSUE THREAD

**Issue by rodehoed** [fatherlinux/ubi-micro#7]
Howdy Scott,

I was playing with the ubi8/9 micro image with docker. And I wanted to add some RPMS and thus using the yum command and specifying the rootfs location. 

But then the yum proces stops at:
`Running scriptlet: glibc-2.28-225.el8.x86_64`
I found out it fails executing a lua script and keeps waiting.

Should this sample still work?

With buildah I can use a workaround en mount the image there. But I would like to this case with docker compose.

**Comment by fatherlinux** [comment]
@rodehoed yeah, I don't think this will work with Docker or DockerCompose. There is no RPM/YUM inside ubi-micro, so you have to use the RPM/YUM located on the host. The only way to do this today is with Buildah. Regrettably, this is the sacrafice for making smaller images.
